packages/terra-demographics-banner/src/DemographicsBannerDisplay.jsx

```diff
@@ -41,9 +45,13 @@ const propTypes = {
    */
   gestationalAge: PropTypes.string,
   /**
-   * Label to display for the GestationalAge
+   * Label to display for the Gestational Age
    */
   gestationalAgeLabel: PropTypes.string,
+  /**
+   * Full text description for the Gestational Age
+   */
+  gestationalAgeFullText: PropTypes.string,
```

Comment:
Why is this opened up as a prop?

Reply:
I think we will need to refactor the demographics banner. When we did this component originally, we wanted to have a version that didn't have translations so that we could test it easier than we did if we required a translation context on the component.

Reply:
Btw, this component isn't designed to be exported by the consumer. It's still expected that they use the DemographicsBanner which contains the translations.